projects/js-packages/partner-coupon/components/redeem-partner-coupon-post-connection/index.jsx

```diff
@@ -0,0 +1,169 @@
+/**
+ * External dependencies
+ */
+import React, { useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+import { ActionButton, JetpackLogo } from '@automattic/jetpack-components';
+import { __, sprintf } from '@wordpress/i18n';
+import cookie from 'cookie';
+
+/**
+ * Internal dependencies
+ */
+import { usePartnerCouponRedemption } from '../../hooks';
+
+/**
+ * Import styles
+ */
+import './style.scss';
+
+export const DISMISS_COOKIE_NAME = 'jp-redeem-partner-coupon-dismissed';
+export const DISMISS_MAX_COOKIE_AGE = 24 * 60 * 60; // 1 day
+
+/**
+ * Is partner coupon redeem CTA dismissed?
+ *
+ * @returns {boolean} Is the redeem CTA dismissed?
+ */
+function isDismissed() {
+	const cookies = cookie.parse( document.cookie );
+	return !! cookies[ DISMISS_COOKIE_NAME ];
+}
+
+/**
+ * Dismiss partner coupon redeem CTA.
+ *
+ * @returns {void}
+ */
+function dismiss() {
+	document.cookie = cookie.serialize( DISMISS_COOKIE_NAME, true, {
+		path: window.location.pathname,
+		maxAge: DISMISS_MAX_COOKIE_AGE,
+	} );
+}
+
+const RedeemPartnerCouponPostConnection = props => {
+	const {
+		connectionStatus,
+		partnerCoupon,
+		assetBaseUrl,
+		siteRawUrl,
+		tracksUserData,
+		analytics,
+	} = props;
+	const [ dismissed, setDismissed ] = useState( isDismissed() );
+
+	const onClick = usePartnerCouponRedemption(
+		partnerCoupon,
+		siteRawUrl,
+		connectionStatus,
+		tracksUserData,
+		analytics
+	);
+
+	const onRemindMeLater = useCallback( () => {
+		dismiss();
+		setDismissed( isDismissed() );
+	}, [ setDismissed ] );
```

Comment:
Actually; would this make sense to track? How often user click the snooze link?

Reply:
I think it would be best if we dedicate time for a separate tracking project/tasks so we have a unified approach rather than doing that ad-hoc in separate patches.
cc @rcoll